src/formatDate.js

```diff
@@ -9,6 +9,41 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+
+  const splitedDate = date.split(oldSeparator);
+
+  const newDate = [];
+
+  for (let i = 0; i < splitedDate.length; i++) {
+    const oldPart = fromFormat[i];
+
+    for (let j = 0; j < splitedDate.length; j++) {
+      const newPart = toFormat[j];
+
+      if (oldPart[0] === newPart[0]) {
+        let newDateElement = splitedDate[i];
+
+        newDateElement = splitedDate[i];
+
+        if (oldPart.length > newPart.length) {
+          newDateElement = newDateElement.split('').slice(-2).join('');
```

Comment:
When converting from 'YYYY' to 'YY', you are correctly taking the last two digits. However, it would be safer to parse the year as an integer before slicing to avoid potential issues with unexpected input formats.